xds/third_party/protoc-gen-validate/import.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Copyright 2018 The gRPC Authors
```

Comment:
should change year to 2020?

Reply:
Good question. I don't know the answer, will keep it as is.